apps/api/src/applicant/ienapplicant.util.service.ts

```diff
@@ -74,8 +75,25 @@ export class IENApplicantUtilService {
       const haPcn = await this.getHaPcn(ha_pcn_id);
       builder
         .innerJoin('ien_applicant_status_audit', 'audit', 'applicant.id = audit.applicant_id')
-        .innerJoin('ien_applicant_status', 'status', 'status.id = audit.status_id')
-        .andWhere(`status.status = 'Applicant Referred to ${haPcn.abbreviation}'`);
+        .innerJoin('ien_applicant_status', 'status', 'status.id = audit.status_id');
+
+      if (filter.recruiter) {
```

Comment:
add to destructuring line?

Reply:
I didn't because I didn't want to make the restructuring line any longer. It was just a baseless feeling.